api/include/opentelemetry/propagators/composite_http_propagator.h

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include "opentelemetry/context/context.h"
+#include "opentelemetry/trace/propagation/httptextformat.h"
+#include "opentelemetry/nostd/span.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace propagators
+{
+    // CompositeHTTPPropagator provides a mechanism for combining multiple
+    // propagators into a single one.
+    template <typename T>
+    class CompositeHTTPPropagator(trace::propagation::HTTPTextFormat<T>) {
```

Comment:
Did you mean

```suggestion
    template <typename T>
    class CompositeHTTPPropagator : public trace::propagation::HTTPTextFormat<T>
```

?